benchmarks/test_basic_parity.py

```diff
@@ -43,20 +43,16 @@ def assert_parity_absolute(pl_values, pt_values, norm_by: float = 1, max_diff: f
 
 # ParityModuleMNIST runs with num_workers=1
 @pytest.mark.parametrize(
-    'cls_model,max_diff_speed,max_diff_memory',
+    'cls_model,max_diff_speed,max_diff_memory,num_epochs,num_runs',
     [
-        (ParityModuleRNN, 0.05, 0.0),
-        (ParityModuleMNIST, 0.25, 0.0),  # todo: lower this thr
+        (ParityModuleRNN, 0.05, 0.0, 4, 3),
+        (ParityModuleMNIST, 0.25, 0.0, 4, 3),  # todo: lower this thr
+        (ParityModuleCIFAR, 4.0, 0.0002, 2, 2),
```

Comment:
```suggestion
        (ParityModuleCIFAR, 0.4, 0.0002, 2, 2),
```
Does that sound as we tolerate 4x slower?

Reply:
It is absolute! It is 4 seconds